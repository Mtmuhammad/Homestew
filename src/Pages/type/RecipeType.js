import{ React, useEffect} from "react";
import { useParams } from "react-router-dom";
import Type from "./Type/Type";

const RecipeType = () => {
  const { type } = useParams();
  
  useEffect(() => {
    window.scrollTo(0, 0)
  }, [])

  return (
    <div
      style={{ maxWidth: "1100px", marginTop: "6rem", marginBottom: "6rem" }}
      className="cuisines container"
    >
      <div className="container">
        <Type type={type} />
      </div>
    </div>
  );
};

export default RecipeType;
