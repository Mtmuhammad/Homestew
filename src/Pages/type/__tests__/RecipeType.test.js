import React from "react";
import { render } from "@testing-library/react";
import RecipeType from "../RecipeType";
import { BrowserRouter } from "react-router-dom";

describe("smoke and snapshot tests", () => {
  test("should render without crashing", () => {
    render(<RecipeType />, { wrapper: BrowserRouter });
  });

  test("should match snapshot", () => {
    const { asFragment } = render(<RecipeType />, { wrapper: BrowserRouter });
    expect(asFragment()).toMatchSnapshot();
  });
});
