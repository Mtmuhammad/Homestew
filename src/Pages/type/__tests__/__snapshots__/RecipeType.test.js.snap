// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`smoke and snapshot tests should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="cuisines container"
    style="max-width: 1100px; margin-top: 6rem; margin-bottom: 6rem;"
  >
    <div
      class="container"
    >
       
      <h3
        class="tertiary"
        style="margin-bottom: 2rem;"
      >
         recipes 
        <i
          class="fa-solid fa-utensils"
          style="color: rgb(0, 128, 128); font-size: 3rem;"
        />
      </h3>
      <div
        class="row"
      >
        <div
          class="spinner-container"
        >
          <div
            class="loading-spinner"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
